### PR TITLE
Don’t cover tests

### PR DESCRIPTION
**Pull request by kinday**
This PR fixes issue #10 and (as an awesome bonus) compel `isparta` to show original ES6 code in reports.

### CODE REVIEW

**Comment by iamstarkov** [comment]
no offense, but i dont like folders in projects’ root

File: app/templates/_package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "<%= moduleDesc %>",
   "main": "index.es5.js",
   "scripts": {
-    "coverage": "isparta cover _mocha index.js -- --require babel/register",
+    "coverage": "babel-node node_modules/.bin/isparta cover _mocha",
```
Comment (iamstarkov) [inline app/templates/_package.json]: you dont need to add `node_modules/.bin`, npm is doing it for you

Comment (kinday) [inline app/templates/_package.json]: Throw error in the air like `babel-node` don’t care!

```
$ npm run coverage

> oboe-promise@0.0.0 coverage /Volumes/Sevilla/Users/kinday/Work/oboe-promise
> babel-node isparta cover _mocha

module.js:338
    throw err;
          ^
Error: Cannot find module '/Volumes/Sevilla/Users/kinday/Work/oboe-promise/isparta'
```

Comment (iamstarkov) [inline app/templates/_package.json]: oh stupid babel-node

**Comment by kinday** [comment]
I played with arguments yesterday but it wasn’t my day at all. I’ll try again and update PR later with flat folder structure.

**Comment by kinday** [comment]
Had no luck. `isparta` seems to ignore (not support) most CLI arguments. `istanbul` has _some_ ES6 support but remains unreliable.

Directory with single file in it looks completely stupid but I prefer it over incorrect coverage report.

File: app/templates/_package.json (hunk 1 above)
Comment (iamstarkov) [inline app/templates/_package.json]: if you run isparta in babel-node, then is it possible to run istanbul itself with babel-node?

Comment (kinday) [inline app/templates/_package.json]: It didn’t work for me. Results in `Transformation error` and `No coverage information was collected`.

Comment (iamstarkov) [inline app/templates/_package.json]: have you seen https://github.com/ambitioninc/babel-istanbul? maybe its more usefull and not that crappy as isparta

**Comment by kinday** [comment]
There’s [long-lasting story](https://github.com/gotwarlost/istanbul/issues/212) about `istanbul` and source maps. Need some time to dive into. If it doesn’t work I’ll try to add support for `-x` flag to `isparta`.

**Comment by iamstarkov** [comment]
did you find smth?

**Comment by kinday** [comment]
Waiting for reaction at gotwarlost/istanbul#463

**Comment by iamstarkov** [comment]
its cool, subscribed to that pull-request. have to note, that tests are red there in node 0.10 https://travis-ci.org/gotwarlost/istanbul/jobs/84845547#L1539

**Comment by iamstarkov** [comment]
see douglasduteil/isparta#94

**Comment by iamstarkov** [comment]
btw, you got some feedback there https://github.com/gotwarlost/istanbul/pull/463

**Comment by iamstarkov** [comment]
fixed in #25

**Pull request closed without merging**